packages/web-components/src/slider/slider.ts

```diff
@@ -12,15 +12,56 @@ import {
   Orientation,
 } from '@microsoft/fast-web-utilities';
 import { getDirection } from '../utils/index.js';
-import { SliderConfiguration, SliderMode, SliderSize } from './slider.options.js';
-import { FormAssociatedSlider } from './slider.form-associated.js';
+import { toggleState } from '../utils/element-internals.js';
+import { SliderConfiguration, SliderMode, SliderOrientation, SliderSize } from './slider.options.js';
 import { convertPixelToPercent } from './slider-utilities.js';
 
+const numberLikeStringConverter: ValueConverter = {
+  fromView(value: string): string {
+    const valueAsNumber = parseFloat(value);
+    return Number.isNaN(valueAsNumber) ? '' : valueAsNumber.toString();
+  },
+  toView(value: any): string | undefined {
+    const valueAsNumber = parseFloat(value);
+    return Number.isNaN(valueAsNumber) ? undefined : valueAsNumber.toString();
+  },
+};
```

Comment:
This is nitpicky, but can we add a quick tsdoc for this so that it's explicit why we're using this as opposed to `nullableNumberConverter`?